libs/api/api.vcxproj

```diff
@@ -175,7 +175,7 @@
     <ProjectReference Include="..\..\rpc_interface\rpc_interface.vcxproj">
       <Project>{1423245d-0249-40fc-a077-ff7780acfe3f}</Project>
     </ProjectReference>
-    <ProjectReference Include="..\..\tools\encode_program_info\encode_program_info.vcxproj">
+    <ProjectReference Include="..\..\tools\export_program_info\export_program_info.vcxproj">
```

Comment:
is there a binary dependency any more?

Reply:
Yes, for bpf2c to generate C code for sample programs in the build step, this exe needs to execute first so that the registry entries are populated correctly.